packages/create-platformatic/test/utils.test.mjs

```diff
@@ -220,15 +220,13 @@ test('isFileAccessible', async ({ end, equal, mock }) => {
   rmSync(tmpDir1, { recursive: true, force: true })
 })
 
-test('getSupportedNodeVersions', async ({ equal, not }) => {
-  const supportedVersions = getSupportedNodeVersions()
-  equal(Array.isArray(supportedVersions), true)
-  not(supportedVersions.length, 0)
+test('minimumSupportedNodeVersions', async ({ equal, not }) => {
+  equal(Array.isArray(minimumSupportedNodeVersions), true)
+  not(minimumSupportedNodeVersions.length, 0)
 })
 
 test('isCurrentVersionSupported', async ({ equal }) => {
```

Comment:
Can we add a test with the case that was failing? If I understand correctly, this is basically the old test (which wasn't enough to spot the problem that this PR is fixing)

Reply:
@marcopiraccini you're right, let me add another test for that case, because previous logic is that if we compare major versions if fails, i.e 16.18.0 should pass but when compared to 18.8.0 it fails because is less than last one.